app/dose-series/dose-series.ts

```diff
@@ -8,6 +8,6 @@ export class DoseSeries {
     id: number;
     med: DoseMedication;
     daysOfWeek: Array<number>;
-    timesOfDay: Array<Date>;
+    timesOfDay: Array<number>; /* [ Date.getTime() ] */
```

Comment:
Server and client are always passing around the number of ms since the epoch, so I had to change this so the right value would be sent.